user_data/strategies/notifier.py

```diff
@@ -1,12 +1,12 @@
 import requests
 import sys
-sys.path.append(Config.EXECUTION_PATH)
+sys.path.append('/root/workspace2/execution')
```

Comment:
please revert

Reply:
ok can you put it in a variable called EXECUTION_PATH in the same class?

Reply:
inside the execution file config it won't work because it needs the path to import Config.EXECUTION_PATH but I can create EXECUTION_PATH variable above sys.path.append so should I do that?

Reply:
yes please, I was referring to notifier.py